### ISSUE TITLE
Alternatives to (planck.io/read-line) in lumo for reading console input.

### ISSUE THREAD

**Issue by abhi18av** [anmonteiro/lumo#222]
Hi @anmonteiro

I was wondering whether something like `(planck.io/read-line)` ships with `lumo.io` to read from console prompts.

I know that perhaps the `nodejs readline module` might be of help but I'm curious about lumo here.

Also, the code completion for ` (require 'lumo. ` fails to show the `io` namespace, but it imports correctly. Is this by design?

**Comment by anmonteiro** [comment]
we'll be adding functions to Lumo's stdlib over time, `read-line` is unfortunately not there.

As for the completion issue, feel free to add the namespaces you're not seeing being auto-completed to [this map](https://github.com/anmonteiro/lumo/blob/5e3e6ed571c445eafafd6cf8bc434dec1dcd3be0/src/cljs/snapshot/lumo/repl.cljs#L1260)

**Comment by abhi18av** [comment]
Yup, done. There's a PR in waiting :)

**Comment by anmonteiro** [comment]
fixed #223

**Comment by abhi18av** [comment]
I'm pasting the implementation of simple repl, for reference of other wanderers.

```clojure
(def readline (js/require "readline"))

(def rl (.createInterface readline
                             (clj->js {:input  (.-stdin  js/process)
                                       :output (.-stdout js/process)})))



(doto rl
      (.setPrompt ">> ")

      (.on "line"
           (fn [line]
             (case line
               "quit" (.close rl)
               ;; default
               (do
                 (println (str "You enter: " line))
                 (.prompt rl)))))
      ;; show info after enter quit
      (.on "close" (fn[]
                     (println "Exit application.")))
      (.prompt))

```
